feature-libs/asm/package.json

```diff
@@ -24,12 +24,14 @@
     "@angular/common": "^13.2.0",
     "@angular/core": "^13.2.0",
     "@angular/forms": "^13.2.0",
+    "@angular/platform-browser-dynamic": "^13.2.0",
```

Comment:
Why do we need this peerDep specifically in ASM package?

Reply:
IMHO those packages (zone and platform browser) should not be listed as peer dependencies of all our libraries. 
Since we're not using those dependencies in the code of our libraries (but only in unit tests), it's not justified to list them as our peer deps. Otherwise, it would affect the resolution of dependencies for customers with unnecessary constrains.

We use those packages only in the setup of unit tests.

Let's fix the script, to ignore our new files `jest.ts`:  https://github.com/SAP/spartacus/blob/8afd1fdb58d4ac1028daedab317463ed8e163d66/tools/config/manage-dependencies.ts#L460-L462

btw. what about renaming those files from `jest.ts` to `test-jest.ts`, so it's obvious that this file is related to bootstraping the tests?

Reply:
I like the idea with re-naming, it's always good to stick to convention. (in this case with karma test.ts files, although it would suggest renaming test to karma-test 🧐 )